app/services/orcid_builder.rb

```diff
@@ -48,6 +48,9 @@ def orcidid(contributor)
     return identifier.uri if identifier.uri
     return identifier.value if identifier.value.start_with?('https://orcid.org/')
 
+    # some records have just the ORCIDID without the URL prefix, add it if so, e.g. druid:tp865ng1792
+    return URI.join('https://orcid.org/', identifier.value).to_s if identifier.source.uri.blank?
```

Comment:
Is there ever a case in which identifier.source.uri isn't `https://orcid.org` or blank? If not, how about getting rid of L52 and replacing L54 with `URI.join('https://orcid.org/', identifier.value).to_s`.

Reply:
That's a good question ... I support in theory it could be "sandbox" for test data, or maybe it might change in the future? Seems unlikely though. 

Our test do verify it works with an alternate URI and I think it was speced this way, but unclear if that's a common situation.

Reply:
Since the rest of `orcid_builder` assumes that you can just used `https://orcid.org` when it is missing, seems fine to always use it.

Reply:
changed (and also in orcid_client PR)

Reply:
Is it ok to put a [HOLD] on this until we've got some clarity around how ORCID identifiers are structured in Cocina?

Reply:
Err, never mind :-)